src/python/WMCore/WorkQueue/WorkQueueReqMgrInterface.py

```diff
@@ -279,8 +282,13 @@ def _workQueueToReqMgrStatus(self, status):
                          'Done' : 'completed'
                          }
         if status in statusMapping:
+            # if wq status passed convert to reqmgr status
             return statusMapping[status]
+        elif status in RequestStatus.REQUEST_STATE_LIST:
```

Comment:
Do we ever pass ReqMgr status to this method? If not, then we do not need to validate against ReqMgr2 statuses.